docs/reference/config.md

```diff
@@ -1144,8 +1144,22 @@ The `manifest` scope allows you to define some meta-data information needed when
 The following settings are available:
 
 `manifest.author`
+: :::{deprecated} 24.09.0-edge
+  Use `manifest.contributors` instead.
+  :::
 : Project author name (use a comma to separate multiple names).
 
+`manifest.contributors`
+: :::{versionadded} 24.09.0-edge
+  :::
+: List of project contributors. Should be a list of maps. The following fields are supported in the contributor map:
+  - `name`: the contributor's name 
+  - `affiliation`: the contributor's affiliated organization
+  - `email`: the contributor's email address
+  - `github`: the contributor's GitHub URL
```

Comment:
```suggestion
  - `github`: the contributor's Git(Hub/Lab etc.) URL
```

Re @pditommaso 's comment, what about this instead to make it clearer it's just the handle you use in any git context? 

Also do you give the full URL or just the handle itself?

Reply:
@bentsherman this is on you

Reply:
Let's keep it as it is for now. I don't typically see people share their gitlab / bitbucket usernames for socials, and in any case the email / affiliation / orcid are the most important. If provenance formats like BCO add other fields or there is a groundswell of user demand, we can add e.g. `gitlab` later.